eth/config.go

```diff
@@ -61,12 +61,12 @@ var DefaultConfig = Config{
 	LightPeers:              100,
 	UltraLightFraction:      75,
 	DatabaseCache:           512,
-	TrieCleanCache:          256,
+	TrieCleanCache:          154,
 	TrieCleanCacheJournal:   "triecache",
 	TrieCleanCacheRejournal: 60 * time.Minute,
 	TrieDirtyCache:          256,
 	TrieTimeout:             60 * time.Minute,
-	SnapshotCache:           256,
+	SnapshotCache:           102,
```

Comment:
`102` looks very specific, I assume it's well backed by scientific research to pinpoint that it's better than `100`

Reply:
256*2/5 :)